android/sdl_android/src/main/java/com/smartdevicelink/transport/TransportManager.java

```diff
@@ -53,20 +53,21 @@
 import com.smartdevicelink.util.DebugTool;
 
 import java.lang.ref.WeakReference;
+import java.util.ArrayList;
```

Comment:
This import is not used.

Reply:
Thanks for the feedback. That was indeed moved to TransportManagerBase...